src/libPMacc/examples/gameOfLife2D/CMakeLists.txt

```diff
@@ -85,8 +85,8 @@ IF(CUDA_SHOW_REGISTER)
 ENDIF(CUDA_SHOW_REGISTER)
 
 IF(CUDA_KEEP_FILES)
-    MAKE_DIRECTORY("${PROJECT_BINARY_DIR}/nvcc_tmp")
-    set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}" --keep --keep-dir "nvcc_tmp")
+    FILE(MAKE_DIRECTORY "${PROJECT_BINARY_DIR}/nvcc_tmp")
```

Comment:
argh, did not see this PR. keep it lower case pls and try to keep the changes minimal